app/views/badges/require.slim

```diff
@@ -3,13 +3,13 @@
     .col-md-12
       .form-container
         h3
-          | OBTER BADGE_
+          | OBTER #{t('badge').upcase.tr('á','Á')}_
```

Comment:
TIL about [this](https://coderwall.com/p/b74yda/rails-upcase-downcase-for-non-ascii-strings)